mypy2junit.py

```diff
@@ -57,13 +58,34 @@ def process_lines(lines: List[str]) -> Tuple[str, bool]:
     return output, int(result['count']) == 0
 
 
+parser = argparse.ArgumentParser()
+parser.add_argument('files', metavar='FILE', nargs='*', help='files to read, if empty, stdin is used')
+parser.add_argument('--output',
+                    type=str,
+                    dest='output',
+                    help='Filename to output to')
+parser.add_argument('--tee', action='store_true')
+
+
 def main():
+    args = parser.parse_args()
+    if args.tee and not args.filename:
```

Comment:
Looks like the `filename` has to be defined as argument as well.

```
Traceback (most recent call last):
  File "mypy2junit.py", line 72, in main
    if args.tee and not args.filename:
AttributeError: 'Namespace' object has no attribute 'filename'
```

Reply:
`filename` should have been `output`. Not sure how I missed that 🤦‍♂️

Reply:
Resolved now though.

Reply:
Great, thank you!